scripts/lib/mkcloud-onhost.sh

```diff
@@ -57,18 +62,18 @@ for port in 22 80 443 5000 7630; do
         host_port_offset=\$(( \$host - \$offset ))
         iptables_unique_rule PREROUTING -t nat -p tcp \\
             --dport \$(( $cloud_port_offset + \$port + \$host_port_offset )) \\
-            -j DNAT --to-destination $net_admin.\$host:\$port
+            -j DNAT --to-destination $net_admin${ip_sep}\$host:\$port
```

Comment:
hmm, this may need to be wrapped because of the :port. So that's a bit problematic